.github/workflows/galaxy-quick.yaml

```
@@ -21,9 +29,10 @@ jobs:
 
   galaxy-quick-test:
     needs: build-artifact
-    secrets: inherit
     uses: ./.github/workflows/galaxy-quick-impl.yaml
     with:
       docker-image: ${{ needs.build-artifact.outputs.dev-docker-image }}
       wheel-artifact-name: ${{ needs.build-artifact.outputs.wheel-artifact-name }}
       build-artifact-name: ${{ needs.build-artifact.outputs.build-artifact-name }}
+      blackhole: ${{ github.event.inputs.bh_glx }}
```

If you decide to go with a dropdown then this condition will need to change, e.g.:
```suggestion
      blackhole: ${{ inputs.arch == 'blackhole' && true || false }}
```

---

No dropdown for now, but it may need to be implemented once we have more BH GLXs in our CI!